Readme.md

```diff
@@ -54,6 +55,17 @@ MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Andr
 ## Contributing
 	  
 Got **something interesting** you'd like to **ask or share**? Start a discussion at `#metabrainz` IRC channel on libera.chat.
+
+## Roadmap
+
+Proposed future plans for the app are as follows:
+
+- Introduce Jetpack Compose to the app.
+- Support more entities for search.
+- Ability to add entities natively from the app.
```

Comment:
MB API does not support it currently so we can't do this.

Reply:
Right. Now that I will be diving into the mb project soon, I think we could add this. But yes, for now let's get rid of this.